### PR TITLE
refactor: change links to api - en

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 1]
```diff
@@ -62,11 +62,11 @@ After all this process, the application returns a response to the request.
 
 1. Create an edge application:
     - On [Console](https://console.azion.com), create an application using the **+ Create** button on the homepage.
-    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create an edge application.
+    - Use the [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) to create an edge application.
     - Use the [Azion CLI](/en/documentation/devtools/cli/create/#edge-applications) to create and deploy an edge application.
 2. Create a domain and associate it with the edge application: 
     - On Console, [create a domain](/en/documentation/products/guides/configure-a-domain/) using the + Create button on the homepage.
-    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create a domain.
+    - Use the [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) to create a domain.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:69]:
```suggestion
    - Use the [Azion API](https://api.azion.com/#a258eabc-b419-47e6-be68-45c2ce0150dd) to create a domain.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:69]: It was the same as create app

File: src/content/docs/en/pages/architectures/edge-application/image-processing.mdx [hunk 2]
```diff
@@ -37,11 +37,11 @@ Azion Image Processor offers simplicity and automation to resource management, a
...
+1. Create an edge application: you must first create the content delivery application layer at the edge of the network. You can do that using [Azion Console](https://console.azion.com), [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) or [Azion CLI](/en/documentation/products/azion-cli/overview/).
     - Choose the [Image Optimization template](https://console.azion.com/create/azion/image-optimization) to accelerate the process.
 2. Create a domain and associate it with the edge application: 
     - On [Console](https://console.azion.com), [create a domain](/en/documentation/products/guides/configure-a-domain/) using the **+ Create** button on the top navbar.
-    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create a domain.
+    - Use the [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) to create a domain.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/image-processing.mdx:44]:
```suggestion
    - Use the [Azion API](https://api.azion.com/#a258eabc-b419-47e6-be68-45c2ce0150dd) to create a domain.
```

File: src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx [hunk 3]
```diff
@@ -51,12 +51,12 @@ After all this process, the content is delivered to the client.
...
 1. Create an edge application:
     - On [Console](https://console.azion.com), create an application using the **+ Create** button on the homepage.
-    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create an edge application.
+    - Use the [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) to create an edge application.
     - Use [Azion CLI] to create and deploy an edge application.
     - Choose the Dynamic and Static File Optimization template.
 2. Create a domain and associate it with the edge application: 
     - On Console, [create a domain](https://www.azion.com/en/documentation/products/guides/configure-a-domain/) using the **+ Create** button on the homepage.
-    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create a domain.
+    - Use the [Azion API](https://api.azion.com/#9c61e0fd-e7aa-4a5f-b29e-7f9d71b98d5f) to create a domain.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/serverless-apps.mdx:59]:
```suggestion
    - Use the [Azion API](https://api.azion.com/#a258eabc-b419-47e6-be68-45c2ce0150dd) to create a domain.
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**